.gitignore

```diff
@@ -239,4 +239,6 @@ _Pvt_Extensions
 
 sample/Sample/out/
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+.gitignore
```

Comment:
can `.idea/` be added to `.gitignore` instead of ignoring `.gitignore`

Reply:
Argh :-) - fixed 👍